RecoDataProducts/inc/KalSeed.hh

```diff
@@ -40,12 +40,12 @@ namespace mu2e {
     UInt_t nTrajSegments() const { return _nseg; }
     bool hasCaloCluster() const { return _chit.caloCluster().isNonnull(); }
     art::Ptr<CaloCluster> const& caloCluster() const { return _chit.caloCluster(); }
-    art::Ptr<HelixSeed> const& helix() const { return _helix; }
```

Comment:
I understand why you removed it; users where using it improperly. BUT I have thinking quite a bit about it and I remembered its real need; I'm using it to associate a reco track with the corresponding Helix that started it. 
Looking at the KalSeedFit_module I don't see a parallel structure of HelixSeedCollection pushed into the event.
Is there a different way I'm not aware of to match a track with the seed Helix?